contributing/README.md-e

```diff
@@ -0,0 +1,140 @@
+Nomad Codebase Documentation
```

Comment:
Is the `README.md-e` name some kind of workaround?

Reply:
Oops - that's due to an `sed` invocation :(